public/docs/ts/latest/cookbook/production-deploy.jade

```diff
@@ -0,0 +1,316 @@
+  it to be imported directly instead of being imported via the Router. 
+
+  It's important to note that your bundling configuration must take lazy loading into consideration.
+  Since there is no ES6 import for that module, normal bundlers don't know that they should also
+  separately bundle to module listed in the router config.
+  You have to manually create additional bundles for each lazy loaded route.
+
+  If you are using Webpack, the 
+  [Angular Ahead-of-Time Webpack Plugin](https://github.com/angular/angular-cli/tree/master/packages/%40ngtools/webpack)
+  will automatically recognize lazy routes.
+
+
+.l-main-section
+:marked
+  ## Server configuration
+
+  Angular apps are 
+  [Single Page Applications](https://en.wikipedia.org/wiki/Single-page_application) (SPAs),
+  which makes them the perfect candidates to be served by simple static HTML server.
+  No preprocessors required!
+
+  There is only one configuration item required of a server hosting an Angular app:
+  it needs to be able to fallback to `index.html` when being asked for a file the server
+  does not have.
+
+  ### Why fallback to `index.html`?
+
+  When using the Angular Router, it is the one in charge of the routing instead of your server.
+  Angular routing defaults to 
+  [HTML 5 pushState](https://angular.io/docs/ts/latest/guide/router.html#!#browser-url-styles),
+  where it manipulates the browser address and history directly.
+  So, while the server has the responsability of serving the files, it is always the Angular app
+  that will determine which routes are valid and construct the page accordingly.
+  
+  Imagine you have the `admin/` route for your `www.my-awesome-app.com` Angular app. 
+  When you visit `www.my-awesome-app.com` most servers will respond with
+  the `index.html` located at `www.my-awesome-app.com/index.html`. 
+  Then you click the link for `admin/` and the your address bar will change 
+  to `www.my-awesome-app.com/admin/` and you're taken to the admin page.
+
+  But if you don't have fallback to `index.html` configured and refresh the page, you will
+  get a `404 - Not Found` page!
+  Which doesn't make a lot of sense since you just saw that the admin page was there.
+
+  When you first clicked the link to `admin/` there was no server request for 
+  `www.my-awesome-app.com/admin/` because Angular manipulated the browser URL directly.
+
+  But when you refresh the `www.my-awesome-app.com/admin/` url, you are not asking *Angular*
+  for the admin route but rather *the server*.
+  And the server does not have a `www.my-awesome-app.com/admin/index.html` to serve, so it instead
+  shows you a 404.
+
+  That is why the server needs to be configured to fallback to `index.html`.
+  When a user asks the server for any route (that isn't a file), you want the server to *always*
+  return default page that boots up Angular.
+  After Angular is loaded and bootstrapped the router will look at the address bar and take
+  the user where he needs to go.
```

Comment:
Should we be avoiding he/she? Either "he" -> "they" or change to "takes the user to the appropriate route".